src/store/rating/api.js

```diff
@@ -1,5 +1,7 @@
 import stelace, { fetchAllResults } from 'src/utils/stelace'
+import pMap from 'p-map'
```

Comment:
pMap is supposed to be used server-side only. You’re breaking compatibility with such imports.

Please check your work in legacy browsers such as IE11.

I had ported this to browser there: https://github.com/stelace/marketplace-demo/blob/dev/src/utils/promise.js.

Reply:
Thanks, I'll make the change.